lib/react.js

```diff
@@ -222,14 +222,19 @@ declare module react {
   declare export var createClass: React$CreateClass;
   declare export function createContext<T>(
     defaultValue: T,
+    calculateChangedBits: ?(a: T, b: T) => number,
```

Comment:
The `createContext` API supports `calculateChangedBits` as an advanced perf optimization technique for libs like Relay and Redux. I'm going to add this and also add a small test case for it too.